osu.Game/Configuration/SessionStatics.cs

```diff
@@ -24,6 +22,18 @@ public void ResetValues()
             ensureDefault(SetDefault<APISeasonalBackgrounds>(Static.SeasonalBackgrounds, null));
         }
 
+        /// <summary>
+        /// Used to revert statics to their defaults after being idle for appropiate amount of time.
+        /// Does not affect loaded seasonal backgrounds.
+        /// </summary>
+        public void ResetValues()
+        {
+            ensureDefault(GetBindable<bool>(Static.LoginOverlayDisplayed));
+            ensureDefault(GetBindable<bool>(Static.MutedAudioNotificationShownOnce));
+            ensureDefault(GetBindable<bool>(Static.LowBatteryNotificationShownOnce));
+            ensureDefault(GetBindable<double?>(Static.LastHoverSoundPlaybackTime));
```

Comment:
Arguably `LastHoverSoundPlaybackTime` also shouldn't be reset. The inactivity period required is long enough to not matter here, but from a semantic point of view there's no reason to do anything with hover sound debounce logic if the user has been deemed inactive.

Interested in more thoughts on this in general from other reviewers.

Reply:
Can agree to this. Would probably also rename this method to be something like `ResetAfterInactivity`.

Reply:
Removed the resetting of `LastHoverSoundPlaybackTime` myself in latest batch of commits.